test/Y2ModuleConfig_test.rb

```diff
@@ -22,7 +43,7 @@
         [
           "audit-laf", "autofs", "ca_mgm", "firstboot", "language", "restore",
           "runlevel", "sshd", "sysconfig", "unknown_profile_item_1",
-          "unknown_profile_item_2", "users"
```

Comment:
In fact, you could use script to configure your system or user-less system as well, or just operated by ssh keys. Expecting that customer must define some section is in fact wrong.